libfwupdplugin/fu-self-test.c

```diff
@@ -1017,15 +1017,15 @@ fu_common_spawn_func(void)
 	guint lines = 0;
 	g_autoptr(GError) error = NULL;
 	g_autofree gchar *fn = NULL;
-	const gchar *argv[3] = {"replace", "test", NULL};
+	const gchar *argv[4] = {"/bin/bash", "replace", "test", NULL};
```

Comment:
Does /bin/sh work as well? If so I think that's preferable - it's a faster launch on a lot of machines.

Reply:
It does, done!